### ISSUE TITLE
LATEST REHLS CRASH WITH 2500 FPS

### ISSUE THREAD

**Issue by 2020karaulov2020** [dreamstalker/rehlds#849]
Crashes sometime with 2500FPS server.

Crash in pthread called by engine(rehlds)
[crash-2021-07-15_07-11-01 (1).log](https://github.com/dreamstalker/rehlds/files/6821950/crash-2021-07-15_07-11-01.1.log)

**Comment by 2020karaulov2020** [comment]
Crashes in 90% times when any player disconnected.

**Comment by 2020karaulov2020** [comment]
Possible problems in FAKE players?

**Comment by hajimura** [comment]
Setting fps to value not more, than 1000 will solve your problem

**Comment by 2020karaulov2020** [comment]
> Setting fps to value not more, than 1000, will solve your problem

No, even with 500 fps i got crash if any Fake players at server

**Comment by fred0r** [comment]
> No, even with 500 fps i got crash if any Fake players at server

what do you mean with 'fake players'?

**Comment by 2020karaulov2020** [comment]
Fake player - bots

**Comment by 2020karaulov2020** [comment]
Fake player bot
